app/lib/account/consent_backend.dart

```diff
@@ -247,10 +247,8 @@ class ConsentBackend {
       () async {
         await action?.onAccept(consent);
         await withRetryTransaction(_db, (tx) async {
-          final c =
-              await tx.lookupValue<Consent>(consent.key, orElse: () => null);
-          if (c == null) return;
-          tx.delete(c.key);
+          final c = await tx.lookupOrNull<Consent>(consent.key);
+          if (c != null) tx.delete(c.key);
```

Comment:
Why not simply `tx.delete(consent.key)` -- also should we not being doing this in the same transaction as everything else done when accepting the invite?

Like maybe the `tx` should be passed to `onAccept`

Reply:
This code predates the retry-datastore-transactions code (hence the outer retry), and it was not merged since. I'll do a follow-up with that.